app/src/main/res/layout/concept_card_example_view.xml

```diff
@@ -1,25 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
-
   <data>
-
     <variable
-      name="subtitledHtml"
-      type="org.oppia.app.model.SubtitledHtml" />
+      name="html"
+      type="CharSequence" />
   </data>
-
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
-
     <TextView
+      android:id="@+id/example"
```

Comment:
Perhaps: concept_card_example_text_view would be clearer? Since Android IDs are global, we should keep them specific.

Reply:
Updated to concept_card_example_text_view.